thrift/events.thrift

```diff
@@ -8,15 +10,15 @@ struct ContentAtomEvent {
    * across all content-atoms of any type (an alternative model might
    * be to have the `type+id` be unique) */
 
-   1: ContentAtomID id
+   1: required ContentAtomID id
 
    // this is a canonical place from which the current version of the
    //content atom's data can be downloaded.
-   2: string url
+   2: required string url
```

Comment:
I'm a bit confused about why we need this `url` field at all. Why would anybody need to download the atom instead of consuming it from the stream? What's the intended use case?